solidity/contracts/bridge/BridgeGovernanceParameters.sol

```diff
@@ -0,0 +1,1710 @@
+
+    event RedemptionTreasuryFeeDivisorUpdateStarted(
+        uint64 newRedemptionTreasuryFeeDivisor,
+        uint256 timestamp
+    );
+    event RedemptionTreasuryFeeDivisorUpdated(
+        uint64 redemptionTreasuryFeeDivisor
+    );
+
+    event RedemptionTxMaxFeeUpdateStarted(
+        uint64 newRedemptionTxMaxFee,
+        uint256 timestamp
+    );
+    event RedemptionTxMaxFeeUpdated(uint64 redemptionTxMaxFee);
+
+    event RedemptionTimeoutUpdateStarted(
+        uint64 newRedemptionTimeout,
+        uint256 timestamp
+    );
+    event RedemptionTimeoutUpdated(uint64 redemptionTimeoutSlashingAmount);
+
+    event RedemptionTimeoutSlashingAmountUpdateStarted(
+        uint96 newRedemptionTimeoutSlashingAmount,
+        uint256 timestamp
+    );
+    event RedemptionTimeoutSlashingAmountUpdated(
+        uint96 redemptionTimeoutSlashingAmount
+    );
+
+    event RedemptionTimeoutNotifierRewardMultiplierUpdateStarted(
+        uint64 newRedemptionTimeoutNotifierRewardMultiplier,
+        uint256 timestamp
+    );
+    event RedemptionTimeoutNotifierRewardMultiplierUpdated(
+        uint64 redemptionTimeoutNotifierRewardMultiplier
+    );
+
+    event MovingFundsTxMaxTotalFeeUpdateStarted(
+        uint64 newMovingFundsTxMaxTotalFeeThreshold,
+        uint256 timestamp
+    );
+    event MovingFundsTxMaxTotalFeeUpdated(uint64 movingFundsTxMaxTotalFee);
+
+    event MovingFundsDustThresholdUpdateStarted(
+        uint64 newMovingFundsDustThresholdThreshold,
+        uint256 timestamp
+    );
+    event MovingFundsDustThresholdUpdated(uint64 movingFundsDustThreshold);
+
+    event MovingFundsTimeoutResetDelayUpdateStarted(
+        uint32 newMovingFundsTimeoutResetDelayThreshold,
+        uint256 timestamp
+    );
+    event MovingFundsTimeoutResetDelayUpdated(
+        uint64 movingFundsTimeoutResetDelay
+    );
+
+    event MovingFundsTimeoutUpdateStarted(
+        uint64 newMovingFundsTimeoutThreshold,
+        uint256 timestamp
+    );
+    event MovingFundsTimeoutUpdated(uint64 movingFundsTimeout);
+
+    event MovingFundsTimeoutSlashingAmountUpdateStarted(
+        uint96 newMovingFundsTimeoutSlashingAmountThreshold,
+        uint256 timestamp
+    );
+    event MovingFundsTimeoutSlashingAmountUpdated(
+        uint96 movingFundsTimeoutSlashingAmount
+    );
+
+    event MovingFundsTimeoutNotifierRewardMultiplierUpdateStarted(
+        uint256 newMovingFundsTimeoutNotifierRewardMultiplierThreshold,
+        uint256 timestamp
+    );
+    event MovingFundsTimeoutNotifierRewardMultiplierUpdated(
+        uint256 movingFundsTimeoutNotifierRewardMultiplier
+    );
+
+    event MovedFundsSweepTxMaxTotalFeeUpdateStarted(
+        uint64 newMovedFundsSweepTxMaxTotalFeeThreshold,
+        uint256 timestamp
+    );
+    event MovedFundsSweepTxMaxTotalFeeUpdated(
+        uint64 movedFundsSweepTxMaxTotalFee
+    );
+
+    event MovedFundsSweepTimeoutUpdateStarted(
+        uint32 newMovedFundsSweepTimeoutThreshold,
```

Comment:
newMovedFundsSweepTimeoutThreshold -> newMovedFundsSweepTime
Similarly, more variables in events in `BridgeGovernanceParameters` have `threshold` added to them.
It's probably not needed. Let's search for `threshold` and remove it when it's not needed.

Reply:
[57954d0](https://github.com/keep-network/tbtc-v2/pull/329/commits/57954d0ae1312b893e46db4a93f3b5e3813dfe02)